src/main/java/core/basesyntax/service/ReadData.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ReadData {

```

Comment:
same, verbs are for methods
```suggestion
public interface FileReader {

```